python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -500,3 +500,18 @@ def test_dask_struct_field_Int_Error(data):
 
     with pytest.raises(IndexError):
         got.struct.field(1000).compute()
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [{}, {}, {}],
+        [{"a": 100, "b": "abc"}, {"a": 42, "b": "def"}, {"a": -87, "b": ""}],
+        [{"a": [1, 2, 3], "b": {"c": 101}}, {"a": [4, 5], "b": {"c": 102}}],
+    ],
+)
+def test_struct_explode(data):
+    expect = pd.DataFrame(data)
```

Comment:
Perhaps we can let the cudf tests worry about cudf-vs-pandas behavior and just compare cudf-vs-dask_cudf in this test?  In other words, lets compare the result of `cudf.Series(data).explode()` to the dask_cudf result (and don't add an explicit `compute` to the dask-cudf case).